app/src/main/res/values-ru/strings.xml

```diff
@@ -49,8 +49,7 @@
     <string name="control_item_2">Чтенеи</string>
     <string name="control_item_3">Расширенные</string>
 
-    <string name="about_us_description">Цель PSLab - создать устройство с открытым исходным кодом (открытое на всех уровнях), которое может быть использовано для экспериментов учителями, студентами и учеными. Наша крошечная карманная лаборатория предоставляет множество датчиков для проведения научных и инженерных экспериментов. Он обеспечивает функции многочисленных измерительных устройств, включая осциллограф, генератор сигналов, частотомер, программируемое напряжение, источник тока и регистратор данных.</string>
-
+    <string name="about_us_description">Цель PSLab - создать устройство с открытым исходным кодом (открытое на всех уровнях), которое может быть использовано для экспериментов учителями, студентами и учеными. Наша крошечная карманная лаборатория предоставляет множество датчиков для проведения научных и инженерных экспериментов. Он обеспечивает функции многочисленных измерительных устройств, включая осциллограф, генератор сигналов, частотомер, программируемое напряжение, источник тока и регистратор данных.\n FOSSASIA - это организация, разрабатывающая приложения с открытым исходным кодом и Open Hardware совместно с мировым сообществом из Азии. Наша цель - предоставить доступ к открытым технологиям, научным приложениям и знаниям, которые улучшают жизнь людей. Мы хотим, чтобы люди могли адаптировать и изменять технологии в соответствии со своими собственными идеями и потребностями, а также проверять науку и знания с помощью подхода открытого доступа.</string>
```

Comment:
you know/speak russian?

Reply:
I used google translate for the translation.